build/Dockerfile

```diff
@@ -1,10 +1,11 @@
+# DO NOT MODIFY. This is produced by template.
```

Comment:
did we add this or is it coming from tooling?

Reply:
our templating script adds it to each file it produces.

Reply:
I see where it's coming from now. Suggest that this should be updated to include the name of the script to run and instead say something like `DO NOT MODIFY BY HAND. Run scripts/new_version.sh to apply any needed modifications`

Reply:
Some of these files are artifacts that end-users may see. I was reluctant to refer to specific build details that won't make any sense to them. Should that be a concern?